admin_panel/config/settings.py

```diff
@@ -0,0 +1,140 @@
+"""
+Django settings for config project.
+
+Generated by 'django-admin startproject' using Django 3.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+
+load_dotenv("../.env")
```

Comment:
Давай как обычно разделим `.env` файлы для конкретного приложения и общий `.env` файл в корне проекта, который используется для докер коспоуз
```suggestion
load_dotenv(".env")
```

Reply:
Тут сейчас правильный путь, корень проекта на один уровень выше, чем сетингс

Reply:
Ну, кстати, да. Мб я ошибся при проверке, но почему-то так не работало. Посмотрю еще раз

Reply:
Ты был прав, поправила. Видимо путь от manage.py считается, а не этого файла